### ISSUE TITLE
4.0.0 Provider Erroring

### ISSUE THREAD

**Issue by chaseconey** [newrelic/terraform-provider-newrelic#2211]
## Overview

When trying to install the latest version of the provider, I am given this error:

```
Error: Failed to install provider

Error while installing newrelic/newrelic v4.0.0: could not query provider
registry for registry.terraform.io/newrelic/newrelic: failed to retrieve
authentication checksums for provider: 404 Not Found
```

When I started working with this a few hours ago it was working. It is no longer.

### Terraform Version
Terraform v0.14.10

### Affected Resource(s)
The provider itself.

### Terraform Configuration
> Please include your `provider` configuration (sensitive details redacted) as well as the configuration of the resources and/or data sources related to the bug report.
```hcl
terraform {
  required_providers {
    newrelic = {
      source = "newrelic/newrelic"
      version = "4.0.0"
    }
  }
}

...


provider "newrelic" {
  account_id = "xxxx"
}
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `rm -rf .terraform`
2. `terraform init`

**Comment by zainul** [comment]
UP

**Comment by bachrilq** [comment]
UP

**Comment by akshaychopra5207** [comment]
facing the same issue at present

**Comment by kidk** [comment]
We had to rollback the release as it wasn't supposed to be a major release. The quick solution is to pin your Terraform to version 3.*

```
terraform {
  required_providers {
    newrelic = {
      source = "newrelic/newrelic"
      version = "~> 3.0"
    }
  }
}
```

Thanks for your patience while we wait on Hashicorp to remove the 4.0 version. This should happen in the next 24 hours.

**Comment by kidk** [comment]
We've just received word from Hashicorp that the 4.0 version has been removed. The workaround is therefore no longer needed.

Thanks all for your patience. I'm going to close this ticket but feel free to continue the conversation.
